PowerEditor/src/WinControls/WindowsDlg/WindowsDlg.cpp

```diff
@@ -228,7 +229,7 @@ BEGIN_WINDOW_MAP(WindowsDlgMap)
 		BEGINCOLS(WRCT_REST,0,0)                       // Begin list control column
 			BEGINROWS(WRCT_REST,0,0)
 				RCREST(IDC_WINDOWS_LIST)
-				RCSPACE(20)
+				RCSPACE(15)

```

Comment:
Restore `RCSPACE(20)` which is not necessary.

Reply:
Restored